inst/@sym/sym.m

```diff
@@ -277,7 +277,6 @@
   end
 
   asm = {};
-  check = true;
```

Comment:
if i remember right this was the reason https://savannah.gnu.org/bugs/?49421, in classdef we can't set properties, so every time when we try to set it we need to run the "Copy" function, other way maybe is store the result in a new var and set it in the end of the script, then we can call the function only one time.